start.sh

```diff
@@ -2,4 +2,5 @@ KALATORI_HOST="127.0.0.1:16726" \
 KALATORI_SEED="" \
 KALATORI_RPC="ws://127.0.0.1:9944" \
 KALATORI_DECIMALS="12" \
+KALATORI_DESTINATION="0xd43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d" \
```

Comment:
is this a well-known account? Good practices require us to post public key or full derivation for this lest we accidentally send something there and it will get stuck.